utils/build/virtual_machine/weblogs/ruby/provision_test-app-ruby-container.yml

```diff
@@ -1,6 +1,6 @@
 weblog:
     name: test-app-ruby-container
-    excluded_os_branches: [amazon_linux2_dotnet6, amazon_linux2_amd64]
+    excluded_os_branches: [amazon_linux2_dotnet6, amazon_linux2_amd64, centos_7_amd64]
```

Comment:
Why are we excluding CentOS from every test?

Reply:
It was failing for all the cases.

Reply:
We still support CentOS 7 so we should definitely try to fix the tests rather than exclude them. Maybe it's related to the mirrors issue that was fixed in https://github.com/DataDog/system-tests/pull/2694

Reply:
We agreed to merge the PR without CentOS to unblock other PRs and that fixing CentOS will be a priority after that